### PR TITLE
Passable lz floodlight

### PR DESCRIPTION
**Pull request by Lengord**
## About The Pull Request
Added passable lz variant of floodlight.
Ps. Irc it vas Vampmare who asked for sprite a year ago. I still have the sprite so why not? 

## Why It's Good For The Game
No more pinky toe injury.

## Changelog

:cl:
add: Passable floodlight
/:cl:

### CODE REVIEW

**Comment by Lengord** [comment]
Без імені

**Comment by JohnDoe50974** [comment]
Vampmare has been gone for much longer than a year though?

**Comment by Lengord** [comment]
Oh, what can i say, time run fast.

File: code/game/machinery/floodlight.dm [hunk 1]
```diff
@@ -130,3 +130,7 @@
 
 	attackby()
 		return
+
+/obj/structure/machinery/floodlight/landing/floor
+	icon_state = "floor_flood01"
+	density = 0
```
Comment (stanalbatross) [inline code/game/machinery/floodlight.dm:136]:
```suggestion
	density = FALSE
```

**Review by stanalbatross — CHANGES_REQUESTED** [review]
You should probably add these to the actual maps as well

**Comment by Lengord** [comment]
> You should probably add these to the actual maps as well

replace on all maps?

**Comment by stanalbatross** [comment]
> > You should probably add these to the actual maps as well
> 
> replace on all maps?

where you see fit, so yeah probably

**Review by Agameofscones — APPROVED** [review]
LGTM

**Comment by Lengord** [comment]
changed floodlites on LZ's, hope nothing is broken

**Comment by Lengord** [comment]
i didn't know such thing as StrongDMM exist, i can discard map changes and redo with StrongDMM. About weird looking stuff, i didn't touch anything it's stock, but can change them if needed?

**Review by stanalbatross — CHANGES_REQUESTED** [review]
strongdmm style must be used for the files, default byond editor bad

**Review by stanalbatross — APPROVED** [review]
code is fine can be put on maps later

**Pull request merged**